apps/system/js/notifications.js

```diff
@@ -228,6 +229,7 @@ var NotificationScreen = {
     this._touchStartX = evt.touches[0].pageX;
     this._touchStartY = evt.touches[0].pageY;
     this._touchPosX = 0;
+    this._touchStartDate = Date.now();
```

Comment:
I'm amazed to find out that touch events really don't have a time component... This seems like a major bit of missing functionality :/